pkgs/top-level/python-packages.nix

```diff
@@ -770,7 +770,9 @@ in {
 
   mkl-service = callPackage ../development/python-modules/mkl-service { };
 
-  monkeyhex = callPackage ../development/python-modules/monkeyhex { };
+  mnist = callPackage ../development/python-modules/mnist { };
+
+  monkeyhex = callPackage ../development/python-modules/monkeyhex { }; 
```

Comment:
I feel like monkeyhex wasn't meant to be added, please remove

Reply:
I accidentally added a space after monkeyhex, hence it was added. I fixed it now.